_overviews/contributors/index.md

```diff
@@ -589,78 +530,28 @@ can browse it at [https://scalacenter.github.io/library-example/](https://scalac
 
 ### Continuous Publication
 
-You need to grant the CI job write access to the Git repository hosting the documentation. This can be achieved
-by creating an SSH key that the CI job can use to push the website to GitHub.
-
-Create an SSH key:
-
-~~~ bash
-$ ssh-keygen -t rsa -b 4096 -C "sbt-site@travis" -f ci/travis-key
-~~~
-
-Make sure to **not** define a passphrase (just leave it empty and press enter), and to add the private
-key (the `ci/travis-key` file) to your `.gitignore`:
+You can extend `.github/workflows/release.yml` to automatically publish documentation to GitHub pages.
+To do so, add another job:
 
-~~~
-ci/secring.asc
-ci/travis-key
-~~~
-
-Add the public key, `ci/travis-key.pub`, in the Deploy Keys section of your GitHub project’s settings page:
-
-![](/resources/images/travis-publishing-key.png)
-
-Make sure you “allow write access” by checking the box.
-
-The private key has to be added to the repository, like we did with the PGP private key. Unfortunately, due
-to a limitation of Travis CI, you can not add several encrypted files. The
-[workaround](https://docs.travis-ci.com/user/encrypting-files/#encrypting-multiple-files) consists in
-creating an archive containing all the files to encrypt. In your case, you want to encrypt the PGP
-key and the SSH key into a single `ci/secrets.tar` file:
-
-~~~ bash
-$ tar cvf ci/secrets.tar ci/secring.asc ci/travis-key
-$ travis encrypt-file ci/secrets.tar ci/secrets.tar.enc --add
-~~~
-
-Make sure to add the `ci/secrets.tar` file to your `.gitignore`:
+```yml
+# .github/workflows/release.yml
+name: #...
```

Comment:
This might be confusing. What do you think of showing the complete workflow file?

Reply:
I can try to be more explicit in the comments but I would prefer showing the complete file: it will be visible in `library-example` and it can be confusing to have a large yml in the middle of the text